### DISCUSSION TITLE
Boilerplate Atomic Design

### DISCUSSION THREAD

**Discussion by virginiarcruz** [frontendbr/forum#1045] [Perguntas e Respostas]
Pessoal,

Vocês utilizam/indicam algum bolierplate pra iniciar o desenvolvimento com Atomic Design?
Basicamente, HTML/SASS/JS, só pra iniciar a componentização.

=)

**Comment by evertondenis** [comment]
Olá @virginiarcruz, o @danilowoz escreveu um artigo e disponibilizou um **boilerplate** sobre o assunto.
Creio que pode lhe ser muito útil: [https://github.com/danilowoz/react-atomic-design](url)

**Comment by virginiarcruz** [comment]
Opa @evertondenis esse ele utiliza react neh.
A princípio queria ver algum que fosse só com o html, sass e js mesmo. Para depois definir qual framework iria utilizar (angular, react, vue).

**Comment by tarciozemel** [comment]
Nunca usei (e parece um pouco abandonado), mas encontrei esse [microscope](https://github.com/afonsopacifer/microscope).
